sklearn/utils/estimator_checks.py

```diff
@@ -2406,7 +2406,7 @@ def check_estimator_sparse_dense(name, estimator_orig):
 
                 assert_equal(probs.shape, (X.shape[0], 4))
         except TypeError as e:
-            if 'sparse' not in repr(e):
+            if 'sparse' not in str.lower(repr(e)):
```

Comment:
I put this because the estimator that used kernel='precomputed' raised something like "Sparse..." which is an OK message